e2e/update-email.spec.ts

```diff
@@ -38,26 +29,43 @@ test.describe('The update-email page', () => {
     );
     await expect(submitButton).toBeVisible();
     await expect(submitButton).toHaveAttribute('type', 'submit');
-  });
 
-  test('The page has sign out button', async () => {
-    const signOutButton = page.getByRole('link', {
-      name: translations.buttons['sign-out']
-    });
+    const signOutButton = page.getByRole('link', { name: 'Sign out' });
 
     await expect(signOutButton).toBeVisible();
     await expect(signOutButton).toHaveAttribute('href', '/signout');
   });
 
-  test('should enable the submit button if the email input is valid', async () => {
+  test('should enable the submit button if the email input is valid', async ({
+    page
+  }) => {
     const emailInput = page.getByLabel(translations.misc.email);
-    const submitButton = page.getByRole('button', {
-      name: translations.buttons['update-email']
-    });
+    const submitButton = page.getByRole('button', { name: 'Update my Email' });
+
     await expect(submitButton).toBeDisabled();
     await emailInput.fill('123');
     await expect(submitButton).toBeDisabled();
     await emailInput.fill('123@gmail.com');
     await expect(submitButton).toBeEnabled();
   });
 });
+
+test.describe('The update-email page when the user is not signed in', () => {
+  test.use({ storageState: { cookies: [], origins: [] } });
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/update-email');
+  });
+
+  test('should redirect to the signin page', async ({ page }) => {
+    if (process.env.CI) {
+      await page.waitForURL('**/learn/');
+    } else {
+      await page.waitForURL('**/learn');
+    }
```

Comment:
```suggestion
    await page.waitForURL(/\/learn\/?/);
```
I'm in two minds about this if this is an improvement, but it is a little more concise :shrug: